docs/concepts.md

```diff
@@ -0,0 +1,95 @@
+<!--[metadata]>
++++
+title = "Machine concepts and help"
+description = "Understand concepts for Docker Machine, including drivers, base OS, IP addresses, environment variables"
+keywords = ["docker, machine, amazonec2, azure, digitalocean, google, openstack, rackspace, softlayer, virtualbox, vmwarefusion, vmwarevcloudair, vmwarevsphere, exoscale"]
+[menu.main]
+parent="workw_machine"
+weight=3
++++
+<![end-metadata]-->
+
+
+# Understand Machine concepts and get help
+
+Docker Machine allows you to provision Docker on virtual machines that reside
+either on your local system or on a cloud provider, and on bare metal servers (physical computers). Docker Machine creates a host on a VM, and you use the Docker Engine client as needed to build images and create containers on the host.
+
+## Drivers for creating machines
+
+To create a virtual machine, you supply Docker Machine with the name of the
+driver you want use. The driver represents the virtual environment. For example,
```

Comment:
I'm not sure this makes sense: "The driver represents the virtual environment."

Maybe something like this? "The driver determines where the virtual machine is created."